src/lib/viewers/controls/color-picker/ColorPickerControl.tsx

```diff
@@ -1,33 +1,32 @@
 import React, { useState } from 'react';
 import classNames from 'classnames';
 import ColorPickerPalette from './ColorPickerPalette';
-import { AnnotationMode } from '../annotations/types';
+import { AnnotationColor } from '../../../AnnotationModule';
 import './ColorPickerControl.scss';
 
+const colors = Object.values(AnnotationColor);
+
 export type Props = {
-    annotationMode?: AnnotationMode;
-    onAnnotationColorClick: (color: string) => void;
+    annotationColor: AnnotationColor;
     isActive?: boolean;
+    onAnnotationColorClick: (color: AnnotationColor) => void;
 };
 
 export default function ColorPickerControl({
-    annotationMode,
+    annotationColor,
     isActive = false,
     onAnnotationColorClick,
     ...rest
 }: Props): JSX.Element | null {
     const [isColorPickerToggled, setIsColorPickerToggled] = useState(false);
 
-    if (annotationMode !== AnnotationMode.DRAWING) {
-        return null;
-    }
-
     return (
         <div className="bp-ColorPickerControl">
             {isColorPickerToggled && (
                 <div className="bp-ColorPickerControl-palette">
                     <ColorPickerPalette
-                        onColorSelect={(color: string): void => {
+                        colors={colors}
+                        onColorSelect={(color): void => {
```

Comment:
Is return type necessary? Should we declare this function outside instead of inline?

Reply:
I don't think the return type is necessary - yes I can declare it outside as it will be a bit more readable.